packages/fxa-settings/src/styles/unit-row.scss

```diff
@@ -9,6 +9,10 @@
     @apply font-header w-full mb-1;
   }
 
+  &-multi-row + &-multi-row {
+    @apply mt-4;
+  }
```

Comment:
Looks like the resulting selector here is `.unit-row-multi-row + .unit-row-multi-row { ... }`. I know this is far more valuable in the CSS than conditionally applying classes in JS, though. I'm just... nervous 😜

Reply:
Yeah, doing it in JS is possible but I'd need to conditionally check the index in `UnitRowSecondaryEmailContentAndActions` and apply `mt-4` if it's not the first index. I chose to go this route because if we ever need multi-rows again, it's more readily available and doesn't have to be applied with JS. I get the concern but I think this is OK personally, it's only two selectors deep and our custom CSS is pretty sparse.

Reply:
Yeah I don't think it needs to be changed. Just gives me the willies ;)